docs/guides/usage.md

```diff
@@ -14,16 +14,17 @@ This guide describes the parts which make up Connect and provides further detail
 
 The following Connect object is the primary interface you will use. All details and additional documentation can be found in [our docs](https://developer.uport.me/uport-connect/reference/index). There is additional helper functions not referenced here, which can also be found in the docs.
 
-* [Connect]() ⇐ <code>[Connect]()</code>
-    * [new Connect(appName, [opts])](#new_Connect_new)
-    * [.requestDisclosure([reqObj], [id])](#Connect+requestDisclosure)
-    * [.getProvider()](#Connect+getProvider) ⇒ <code>UportSubprovider</code>
-    * [.contract(abi)](#Connect+contract) ⇒ <code>Object</code>
-    * [.createVerificationRequest(reqObj, [id])](#Connect+createVerificationRequest)
-    * [.attest([credential], [id])](#Connect+attest)
-    * [.send(request, id, [opts])](#Connect+send)
-    * [.onResponse(id, cb)](#Connect+onResponse) ⇒ <code>Promise.&lt;Object, Error&gt;</code>
-    * [.logout()](#Connect+logout)
+  * [Connect](#Connect)
+      * [new Connect(appName, [opts])](#new_Connect_new)
```

Comment:
Not clear how this list of functions is ordered.  I would suggest either ordering them by expected frequency of usage / commonality and related features, or by alphabetically.

Reply:
yeah I can see how jsdocs orders them, but not worth manually ordering at the time